src/plots/plots.js

```diff
@@ -41,7 +41,7 @@ plots.fontWeight = 'normal';
  */
 plots.register = function(_module, thisType, categoriesIn, meta) {
     if(modules[thisType]) {
-        console.warn('type ' + thisType + ' already registered');
+        console.log('type ' + thisType + ' already registered');
```

Comment:
Bringing in @bpostlethwaite and @alexcjohnson 

Should we warn (or spit out a console.log) when users register the same plot module twice?

I'd vote for doing nothing at all.

@mdtusz If I'm no mistaken, every time a user registers two or more trace modules that rely on the same plot module e.g. 

```js
var PlotlyBar = require('plotly.js/lib/bar');
var PlotlyBox = require('plotly.js/lib/box');

Plotly.register([PlotlyBar, PlotlyBox]);
```

would split out a log.